### PR TITLE
chore(issues): Update CODEOWNERS to reflect reality

### PR DESCRIPTION
**Pull request by mrduncan**
This should be the last major pass at changes relating to the issues team.

Please call out anything which doesn't look right!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -329,16 +329,16 @@ static/app/components/events/eventStatisticalDetector/                    @getse
 /static/app/views/organizationIntegrations                           @getsentry/product-owners-settings-integrations
 /static/app/views/settings/project/projectOwnership/                 @getsentry/issues
 
-/src/sentry/digests/                                                 @getsentry/issues
-/src/sentry/identity/                                                @getsentry/issues
+/src/sentry/digests/                                                 @getsentry/alerts-notifications
+/src/sentry/identity/                                                @getsentry/ecosystem
```
Comment (scttcper) [inline .github/CODEOWNERS:333]: is ecosystem being used again?

**Review by scttcper — APPROVED** [review]
interesting okay https://github.com/orgs/getsentry/teams/ecosystem

File: .github/CODEOWNERS (hunk 1 above)
Comment (sentaur-athena) [reply .github/CODEOWNERS:333]: Ecosystem is back but we don't own identity. I don't know what's the best way to handle this so tagging @leedongwei 

Maybe keep it with old enterprise so it gets routed to the rotation?

Comment (mrduncan) [reply .github/CODEOWNERS:333]: Good call, that also keeps it consistent with the `/src/sentry/models/identity.py` line below which is also currently routed to enterprise

**Pull request merged**